packages/vendure-plugin-accept-blue/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.6.2 (2024-09-18)
+
+- Fix #452
```

Comment:
Can you make this a bit more descriptive, so that its also clear what is fixed without reading the issue?